common/openshift.init.yml

```diff
@@ -108,6 +114,8 @@ objects:
       database-user: ${NAME}  
       oracle-user: ${ORACLEDB_USER}
       oracle-password: ${ORACLEDB_PASSWORD}
+      oracle-user-write: ${ORACLEDB_USER_W}
```

Comment:
Those two can be removed, as they are redundant. We can make use of just the **oracle-user** and **oracle-password**